File: solucoes/2021/nivel jr/fase 3/js/plano.js
var M, N, D

scanf("%d", "M")
scanf("%d", "N")

var vagas_d = []
var vagas_pegas = []

for(var i = 0; i < N; i++){
    scanf("%d", "D")
    vagas_d.push(D)
}


for(var i = 0; i < N; i++){
    if(!vagas_pegas.includes(vagas_d[i])){
        vagas_pegas.push(vagas_d[i])
    }
}


printf("%d", vagas_pegas.length)

File: solucoes/2021/nivel jr/fase 3/py/plano.py
M = int(input())
N = int(input())
vagas_d = [int(input()) for _ in range(N)]
vagas_pegas = []

for i in range(N):
    if vagas_d[i] not in vagas_pegas:
        vagas_pegas.append(vagas_d[i])

print(len(vagas_pegas))
